install.sh

```diff
@@ -96,6 +96,24 @@ LOGS_INSTALL_LOG_FILE_PATH="$LOGS_DIR_PATH/$LOGS_INSTALL_LOG_FILE_NAME"
         echo "Not detected package manager. Driver may not work properly because required packages have not been installed. Please create an issue (https://github.com/asus-linux-drivers/asus-numberpad-driver/issues)."
     fi
 
+    # https://github.com/asus-linux-drivers/asus-numberpad-driver/pull/255
+    if [ "$DESKTOP_SESSION" == "plasma" ]
+    then
+        qdbus=$(which qdbus)
+        plasma_ver=$(plasmashell --version | grep -oE '[0-9]+' | head -1)
+
+        if [ -n "$qdbus" ]
+        then
+            if [ "$plasma_ver" -eq 5 ]; then
+                sudo pacman -S --noconfirm --needed qt5-tools
+            elif [ "$plasma_ver" -eq 6 ]; then
```

Comment:
Plasma 5 isn't even in Arch Linux repos, so I don't think it's wise to support old installations, as Arch is a rolling release distro.